storybook/decorators/with-max-width-wrapper.js

```diff
@@ -3,15 +3,12 @@
  */
 import styled from '@emotion/styled';
 
-/**
- * A Storybook decorator to wrap a story in a div applying a max width and
- * padding. This can be used to simulate real world constraints on components
- * such as being located within the WordPress editor sidebars.
- */
-
-const Wrapper = styled.div`
-	max-width: 248px;
-`;
+const maxWidthWrapperMap = {
+	none: 0,
+	'wordpress-sidebar': 248,
+	'large-container': 960,
+	'small-container': 600,
```

Comment:
The only thing I'd do would be to change the order here, so they go from small to large:

```suggestion
	'small-container': 600,
	'large-container': 960,
```